runtime/drivers/pinot/pinot.go

```diff
@@ -13,6 +13,7 @@ import (
 
 func init() {
 	drivers.Register("pinot", driver{})
+	drivers.RegisterAsConnector("pinot", driver{})
```

Comment:
this is required for it to be in list of connectors for display in UI ?

Reply:
Yes, it's there in clickhouse connector as well.